From: Tzafrir Cohen <tzafrir@debian.org>
Subject: bashism in the test script

--- a/test/test1
+++ b/test/test1
@@ -11,7 +11,7 @@
 DIR=test
 
 TMPLOG=/tmp/test1.log
-function dotest(){
+dotest(){
 	# Run the given "ispell -a"-like command and expect all the words to
 	# be correct, or all incorrect, as requested.
 	local TESTNAME="$1"
@@ -50,7 +50,7 @@
 }
 
 TMPFILE=/tmp/test1.in
-function test_all(){
+test_all(){
 	TESTNAME="$1"
 	shift
 	>$TMPFILE
@@ -64,7 +64,7 @@
 			dotest "$TESTNAME/$section" "$TMPFILE" "$SECHEAD" "$@"
 			>$TMPFILE
 			SECHEAD="$line"
-			let section++
+			section=$((section+1))
 			;;
 		*)
 			echo "$line" >>$TMPFILE
