Description: Convert WHATSNEW file to UTF-8
Author: Lior Kaplan <kaplan@debian.org>

--- a/WHATSNEW
+++ b/WHATSNEW
@@ -12,7 +12,7 @@
    and hspell_trycorrect() to core-dump on an empty word.
 
  * Fixed a bug which caused us to accept a definite article on verbs with
-   objects, e.g., äëåáùï.
+   objects, e.g., ×”×›×•×‘×©×Ÿ.
 
  * The "myspell" target was removed, as this format has been all but abandoned
    in recent years. Use "make hunspell" instead.
@@ -23,7 +23,7 @@
    hunspell-format dictionaries.
 
  * In previous releases, generated hunspell and aspell dictionaries incorrectly
-   accepted bare infinitive verbs, e.g., éùåï. This is fixed in this release
+   accepted bare infinitive verbs, e.g., ×™×©×•×Ÿ. This is fixed in this release
    for hunspell dictionaries, using the hunspell-specific NEEDAFFIX feature.
    Unfortunately, aspell does not have this feature, so the aspell dictionary
    still has this bug. Added a "make test" to test this bug.
@@ -302,7 +302,7 @@
    In order to build the source rpm you must --define in the command-line either
    'fat 0' or 'fat 1'. Can anyone tell me how can I set a default for that?
 
- * Infinitives now allow áëìî prefixes, not just ì: ìùáåø, áùáåø, ëùáåø, îùáåø.
+ * Infinitives now allow ×‘×›×œ× prefixes, not just ×œ: ×œ×©×‘×•×¨, ×‘×©×‘×•×¨, ×›×©×‘×•×¨, ××©×‘×•×¨.
 
  * Added a "-d" option to the word-list generators, which prints derivation
    explanation for all inflections, and also gender information for all nouns.
@@ -334,9 +334,9 @@
    option not working.
    Thanks to Tzafrir Cohen and Oron Peled for the initial version of the spec.
 
- * Shem Poal Natuy + Kinuy for all verbs, things like: åúáùá ,éğğèöäá
+ * Shem Poal Natuy + Kinuy for all verbs, things like: ×•×ª×‘×©×‘ ,×™× × ×˜×¦×”×‘
    There is probably an error in the conjugation of these Shmot Poal from Binyan
-   Qal. woo creates áæåëøğå, while the strict rules say it should be áæëøğå.
+   Qal. woo creates ×‘×–×•×›×¨× ×•, while the strict rules say it should be ×‘×–×›×¨× ×•.
    This would be settled by the next release.
 
  * Added "-i" option, to be used in addition to "-a" from within other
@@ -379,7 +379,7 @@
    Read http://www.math.tau.ac.il/~dekelts/lyx/instructions2.html for
    Hebrew LyX general instructions.
 
- * No longer incorrectly recognize ùìåååí (should be ùìååí, "that to their
+ * No longer incorrectly recognize ×©×œ×•×•×•× (should be ×©×œ×•×•×, "that to their
    hook").
 
  * The wolig.pl automatic inflector can handle country names, and many more
